### DISCUSSION TITLE
📔【计算机网络】TCP 的流量控制与拥塞控制 🆕

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#90] [Announcements]
https://imageslr.github.io/2020/tcp-flow-control-congestion.html 

 更多面试题总结请看：🗂【面试题】技术面试题汇总

**Comment by wangx26** [comment]
”超时重传1.把 sshthresh 设为原来的一半（乘性减）“应为设置为出现拥塞时窗口的一半

**Comment by imageslr** [comment]
> @wangx26
> ”超时重传1.把 sshthresh 设为原来的一半（乘性减）“应为设置为出现拥塞时窗口的一半

sshthresh = cwnd / 2，而不是 sshthresh /= 2？这个我没确认过。

不过如果这样的话，那“cwnd 重置为 sshthresh”之后，cwnd 就没法加性增了吧？

---

update：感谢反馈
